src/coordinate_transform.rs

```diff
@@ -0,0 +1,176 @@
+#[cfg(feature = "tie-points")]
+use std::rc::Rc;
+
+#[cfg(feature = "tie-points")]
+use geo_index::rtree::OwnedRTree;
+use geo_types::Coord;
+use tiff::{TiffError, TiffFormatError, TiffResult};
+
+#[cfg(feature = "tie-points")]
+use crate::coordinate_transform::tie_points::Face;
+
+mod affine_transform;
+mod tie_point_and_pixel_scale;
+#[cfg(feature = "tie-points")]
+mod tie_points;
+
+const MODEL_TIE_POINT_TAG: &str = "ModelTiePointTag";
+const MODEL_PIXEL_SCALE_TAG: &str = "ModelPixelScaleTag";
+const MODEL_TRANSFORMATION_TAG: &str = "ModelTransformationTag";
+
+#[derive(Debug)]
+pub enum CoordinateTransform {
```

Comment:
Document this by adding a link to https://docs.ogc.org/is/19-008r4/19-008r4.html#_raster_to_model_coordinate_transformation_requirements?

Reply:
Hmm, I haven't seen any new commits yet, did you do a `git push`?

Reply:
I got interrupted today, sorry. Pushed what I've done so far.